src/components/participants/participant-image.tsx

```diff
@@ -0,0 +1,30 @@
+import Image from "next/image"
+
+type ParticipantImageProps = {
+  src: string
+  alt: string
+  shouldHasLine: boolean
+}
+
+const afterClasses =
+  " after:content-[''] after:h-[1px] after:width-1/2 after:right-1 after:w-full after:bg-primary-100 after:absolute after:top-1/2 relative after:translate-x-1/2 after:left-1/2"
+
+export function ParticipantImage({
+  src,
+  alt,
+  shouldHasLine,
```

Comment:
We could name this as `lined` or `bordered` instead of `shouldHasLine`. What do you think?

Reply:
No problem. I prefer `lined` xD